problem_builder/public/js/problem_builder.js

```diff
@@ -0,0 +1,40 @@
+window.ProblemBuilder = {
+
+    transformClarifications: function(element) {
+        element = $(element);
```

Comment:
@bradenmacdonald Yeah, `var $element = $(element)` makes it more clear. The problem that I've run into is that xblock JS init functions are not invoked consistently in Studio/LMS/workbench. Sometimes they are initialized with a jquery object and sometimes with a plain DOM node. While `transformExisting` works with either, the mutation observer requires a plain DOM node, so the `var $element = $(element)` is just to ensure that `$element` is of consistent type.

This should probably be fixed in Studio/LMS to make xblock initialization consistent, but I didn't have time to debug this further.